### PR TITLE
Update 0.7

### PR DESCRIPTION
**Pull request by abelsiqueira**
Tentative solution to https://discourse.julialang.org/t/null-function-pointer-in-a-function-creaded-with-eval/13202/3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,8 +8,17 @@ julia:
   - 0.4
   - 0.5
   - 0.6
```
Comment (dpo) [inline .travis.yml]: Thanks. Let's drop 0.4 and 0.5. Registered packages are now required to support 0.6 or higher, but not previous versions.

File: src/amd_functions.jl [hunk 2]
```diff
@@ -1,9 +1,11 @@
-global const _libamd = Libdl.dlopen("libamd")
+function __init__()
```
Comment (dpo) [inline src/amd_functions.jl]: Interesting! How on Earth did you figure that out?

Comment (abelsiqueira) [reply src/amd_functions.jl]: `_libamd` was null from the start, so the problem was in this line. Then I remembered `__init__`, which is kind of a Module constructor, which runs at runtime. However, I can't explain fully why this works, and why it wasn't needed before. I'm guessing it has to do with the compilation of Libdl which is a separate package in 0.7.

File: .travis.yml (hunk 1 above)
Comment (abelsiqueira) [reply .travis.yml]: Done

**Comment by Xx-fractale-xX** [comment]
I found the (old) problem on Windows 64 bits.
Cint and Clong are both Int32. 
In the documentation, I only find Clonglong (Int64), maybe they change the syntax?
https://docs.julialang.org/en/stable/manual/calling-c-and-fortran-code/index.html
cint clong windows

File: src/amd_functions.jl (hunk 2 above)
Comment (dpo) [reply src/amd_functions.jl]: fantastic! Thank you!

**Comment by abelsiqueira** [comment]
Thanks @Xx-fractale-xX, that did it.

**Comment by Xx-fractale-xX** [comment]
Great! And issue #9 is also resolved.

**Comment by dpo** [comment]
Excellent! Thank you both!

**Pull request merged**